examples/website/geospatial/examples.ts

```diff
@@ -76,6 +76,80 @@ export const EXAMPLES: Record<string, Record<string, Example>> = {
           properties
         };
       }
+    },
+    'Major rivers (gzip)': {
+      format: 'geoparquet',
+      data: 'https://raw.githubusercontent.com/visgl/deck.gl-data/master/formats/geoparquet/major-rivers/major-rivers_0.4.0_gzip.parquet',
+      viewState: {
+        height: 600,
+        width: 800,
+        pitch: 45,
+        maxPitch: 60,
+        bearing: 0,
+        minZoom: 1,
+        maxZoom: 30,
+        zoom: 1.76,
+        longitude: -4.65,
+        latitude: -29.76
+      },
+      getTooltipData: function ({object}) {
+        const {NAME, ...properties} = object?.properties || {};
+        return {
+          title: NAME as string,
+          properties
+        };
+      }
+    },
+    'Fort Collins streets (brotli)': {
+      format: 'geoparquet',
+      data: 'https://raw.githubusercontent.com/visgl/deck.gl-data/master/formats/geoparquet/fort-collins-streets/fort-collins-streets_0.4.0_brotli.parquet',
+      viewState: {
+        height: 600,
+        width: 800,
+        pitch: 45,
+        maxPitch: 60,
+        bearing: 0,
+        minZoom: 1,
+        maxZoom: 30,
+        zoom: 13,
+        longitude: -105.073,
+        latitude: 40.542
+      },
+      getTooltipData: function ({object}) {
+        const {STRNAME, ...properties} = object?.properties || {};
+        return {
+          title: STRNAME as string,
+          properties
+        };
+      }
+    },
+    'Fort Collins address (no_compression)': {
```

Comment:
Is this the same dataset with different compression? If so align the names. If not, this is fine.

Reply:
It is different